### DISCUSSION TITLE
Are there additional instructions to download ljs Speech on Windows VSCode?

### DISCUSSION THREAD

**Discussion by RoakyWood** [coqui-ai/TTS#3978] [General Q&A]
Up to this step.

Is there some additional file structure that needs to be created?

python -c 'from TTS.utils.downloaders import download_ljspeech; download_ljspeech("../recipes/ljspeech/");'  
  File "<string>", line 1
    from TTS.utils.downloaders import download_ljspeech; download_ljspeech(../recipes/ljspeech/);
                                                                           ^
SyntaxError: invalid syntax

**Comment by eginhard** [comment]
The double quotes get removed somehow, probably a Windows thing... Maybe escape them like this: `python -c 'from TTS.utils.downloaders import download_ljspeech; download_ljspeech(\"../recipes/ljspeech/\");'` Or open a Python shell and then run that command.
